src/Uno.Templates/content/unomauilib/.template.config/template.json

```diff
@@ -112,25 +112,7 @@
         ]
       }
     },
-    "libraryBaseTargetFramework": {
-      "type": "generated",
-      "generator": "switch",
-      "replaces": "$libraryBaseTargetFramework$",
-      "parameters": {
-        "evaluator": "C++",
-        "datatype": "string",
-        "cases": [
-          {
-            "condition": "(tfm == 'net7.0')",
-            "value": "net7.0"
-          },
-          {
-            "condition": "(tfm == 'net8.0')",
-            "value": "net8.0"
-          }
-        ]
-      }
-    },
+    // TODO: Should we be using $(DotNetVersion) instead of explicitly defining each tfm?
```

Comment:
@dansiegel this is w.r.t the mobileTargetFrameworks for the maui library template - do we want to assume the presence of $DotNetVersion?